### ISSUE TITLE
a question about selection

### ISSUE THREAD

**Issue by SydCS** [wx-zhang/spu#5]
Hello!
In https://github.com/wx-zhang/spu/blob/main/trainer/spu.py#L76, you calculate the L2-Norm of logits_per_image as loss. I wonder why it works and what's its connection with real loss function.
Thank u for ur response.
